package.json

```diff
@@ -2138,7 +2138,7 @@
     },
     "devDependencies": {
         "@johnnymorganz/stylua-bin": "^0.20.0",
-        "@types/lodash-es": "^4.17.12",
+        "@types/lodash": "^4.17.4",
```

Comment:
I explained this in the commit message, but I replaced this dependency because we actually compile our TS down to JS that uses commonjs anyway.

Importing the ESModule from a unit test proved troublesome (I was getting the below error). Honestly, I'm surprised this worked at runtime at all, given we weren't using commonjs, but the JS import system is admittedly not my most well-studied area.

```
> vscode-neovim@1.15.0 test:unit
> mocha --exit ./out/test/unit/*.js ./out/test/unit/**/*.js


 Exception during run: Error [ERR_REQUIRE_ESM]: require() of ES Module /home/nick/Documents/code/vscode-neovim/node_modules/lodash-es/lodash.js from /home/nick/Documents/code/vscode-neovim/out/status_line/status_line_message_timer.js not supported.
Instead change the require of lodash.js in /home/nick/Documents/code/vscode-neovim/out/status_line/status_line_message_timer.js to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (/home/nick/Documents/code/vscode-neovim/out/status_line/status_line_message_timer.js:4:21)
    at Object.<anonymous> (/home/nick/Documents/code/vscode-neovim/out/test/unit/status_line/status_line_message_timer.test.js:28:37)
    at async formattedImport (/home/nick/Documents/code/vscode-neovim/node_modules/mocha/lib/nodejs/esm-utils.js:9:14)
    at async exports.requireOrImport (/home/nick/Documents/code/vscode-neovim/node_modules/mocha/lib/nodejs/esm-utils.js:42:28)
    at async exports.loadFilesAsync (/home/nick/Documents/code/vscode-neovim/node_modules/mocha/lib/nodejs/esm-utils.js:100:20)
    at async singleRun (/home/nick/Documents/code/vscode-neovim/node_modules/mocha/lib/cli/run-helpers.js:125:3)
    at async exports.handler (/home/nick/Documents/code/vscode-neovim/node_modules/mocha/lib/cli/run.js:370:5) {
  code: 'ERR_REQUIRE_ESM'
}
```

If there's a better way to solve this, do let me know, but I think this is correct.